lib/assessment.md

```diff
@@ -138,7 +176,16 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+.divOne {
+height: 50px;
+width: 50px;
+float: right;
+}
+.divFour {
+  height: 50px;
+  width: 50px;
+  clear: left;
+}
```

Comment:
I think you may have misunderstood this question - we were looking for a description of the layout of the `div` elements in the HTML that was created by the given CSS.